api/http.yaml

```diff
@@ -23,7 +23,6 @@ components:
             $ref: "#/components/schemas/HTTPHeader"
         body:
           type: string
-          format: byte
```

Comment:
Basically, this means the `body` should be in `base64` format. But this is not true. I removed this so prism could add the correct validations to the requests I'm sending.

Reply:
good catch